presto-spark-base/src/test/java/com/facebook/presto/spark/execution/DetachedNativeExecutionProcess.java

```diff
@@ -76,10 +77,7 @@ public void start() throws ExecutionException, InterruptedException
     @Override
     public int getPort()
     {
-        String configuredPort = System.getProperty("NATIVE_PORT");
-        if (configuredPort != null) {
-            return Integer.valueOf(configuredPort);
-        }
-        return DEFAULT_PORT;
+        String configuredPort = requireNonNull(System.getProperty("NATIVE_PORT"), "NATIVE_PORT not set for interactive debugging");
```

Comment:
same

Reply:
When this class is used, NATIVE_PORT has to be set. Otherwise there is something wrong.